Sources/Html/Attributes.swift

```diff
@@ -293,6 +293,19 @@ public func draggable<T>(_ value: Draggable) -> Attribute<T> {
   return .init("draggable", value == .auto ? nil : value.rawValue)
 }
 
+public enum Enctype: String {
+  case formURLEncoded = "application/x-www-form-urlencoded"
+  case multipartFormData = "multipart/form-data"
+  case plainText = "text/plain"
```

Comment:
And this line would be:
```suggestion
  case textPlain = "text/plain"
```